src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx

```diff
@@ -57,14 +57,6 @@ const AssetList: FunctionComponent<AssetListProps> = ({
       label,
     }
 
-    if (isReported && icon?.reportedIconSvg) {
-      return {
-        ...baseItem,
-        iconUrl: icon ? icon.reportedIconSvg : '',
-        isReported: true,
```

Comment:
The `isReported` variable is still used on lines 47 and 63. Can't it be removed?